src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days <= 2) {
+    result = days * 40;
+  }
+
+  if (days < 7 && days >= 3) {
+    result = days * 40 - 20;
+  }
```

Comment:
```suggestion
  if (days < 7) {
    result = days * 40 - 20;
  }
```
Ты в предыдущем if уже проверил что `days >= 2`, если ты там будешь сразу возвращать нужное значение, то в этом if можно будет не писатьт >=3

```
  if (days <= 2) {
    return days * 40;
  }
```
вот так

Reply:
Виправив.